apps/i18n/poetry/en_us.json

```diff
@@ -0,0 +1,29 @@
+{
+  "deletedSprite": "It looks like you removed a sprite. You can add it back from the **Sprites** toolbox.",
+  "effectWithEvent": "Try using a new foreground or background effect under your `when line shows` event block.",
+  "makeBackground": "You need to add a background effect.",
+  "makeForeground": "You need to add a foreground effect.",
+  "noBehavior": "Click on the **Sprites** toolbox to find this block.\n<xml><block type=\"PoemBot_startBehavior\"><title name=\"ANIMATION_NAME\">\"tropicalfish_04_1\"</title><title name=\"BEHAVIOR\">swimming_left_and_right</title></block></xml>",
+  "noTextEffects": "You need to add a text effect to your project.",
+  "oneTextEffect": "Great work! You added one text effect. Try adding another.",
+  "removedSprite": "It looks like you removed a sprite. You can add it back from the **Sprites** toolbox.",
+  "setSize": "Use a `set size` block to change the size of your sprite. Use a number other than 100.",
+  "spriteEvent": "Try adding or removing a sprite under your `when line shows` event block.",
+  "successAddedSprite": "Great work adding a new sprite with an event! Click “Keep Playing” to try removing a sprite with an event.",
+  "successBackgroundEvent": "Great job changing your background with an event! Click “Keep Playing” to try changing the foreground effect.",
+  "successEffectWithEvent": "Congratulations, you added an effect to your code! You can click 'Keep playing' to add even more effects.",
+  "successForegroundEvent": "Great job changing your foreground with an event! Click “Keep Playing” to try changing the background effect.",
+  "successLevel1": "Congratulations! You completed Level 1.",
```

Comment:
sorry i missed this before, but unrelated super nit: these success messages could be dynamic

```javascript
// in poetry/en_us.json
"successLevelN": "Congratulations! You completed Level {n}."
```

```javascript
// usage in code/levelbuilder
i18n.successLevelN({n: 1})
```

we could also move this to common/en_us.json because this seems like a useful utility string for levelbuilders, although that makes it muuuuuch less discoverable. just some thoughts -- not a blocker for this change

Reply:
That would also require changing the way we pass through the i18n keys from validation code, so I'll leave it as future work for now. I'm also not sure whether these are finalized strings or just placeholders